lib/cc/cli/analyze.rb

```diff
@@ -1,11 +1,9 @@
-require "securerandom"
-
 module CC
   module CLI
     class Analyze < Command
       include CC::Analyzer
 
-      def initialize(args = [])
+      def initialize(_args = [])
```

Comment:
It was a drive-by fix: `args` is unused, if this were written today, it would be a rubocop issue.